x-pack/solutions/security/plugins/security_solution/scripts/endpoint/common/elastic_agent_service.ts

```diff
@@ -178,6 +178,6 @@ const getElasticAgentDockerArgs = ({
     '--env',
     'FLEET_INSECURE=true',
     '--rm',
-    `docker.elastic.co/beats/elastic-agent:${agentVersion}`,
```

Comment:
A quick search of `docker.elastic.co/beats/` shows other code instances that should also be updated.

Reply:
Correct, not under our plugin but in other plugins. I'm guessing this URL change is relevant throughout the codebase?

Reply:
I don't know that... Just did the search and it seems that most are under fleet. I'll let them know in the thread we have going with them in slack

Reply:
I think we also should update `x-pack/test/osquery_cypress/agent.ts` in this PR.

Reply:
ahhh. yes... forgot about OSQuery. just made the change and pushed it